BILLthebuilder/automart
# #166577104 Add validations for POST api endpoints

#### What does this PR do?
* It validates all the api endpoints for correct data
#### Description of Task to be completed?
* Makes sure requests are not empty
* Ensures the correct value is put in the requests
#### How should this be manually tested?
* Clone the repository
* Checkout to the `bg-validations-166577104`branch
* Clone the repo
* Run `npm install` to install package dependencies
* Run `npm test` to run tests
* All the tests should pass
* Run `npm start`
* Test all the api endpoints defined in the readme.md on develop branch on postman
* Everything should work as expected with no errors
#### What are the relevant pivotal tracker stories?
* [#166577104](https://www.pivotaltracker.com/story/show/166577104 )
#### Screenshots
* none
